packages/sdk/src/index.ts

```diff
@@ -14,9 +14,9 @@ export {
   defaultLibp2p,
   createLibp2pAndUpdateOptions
 } from "./create/index.js";
-export { wakuLightPush } from "./protocols/light_push.js";
-export { wakuFilter } from "./protocols/filter/index.js";
-export { wakuStore } from "./protocols/store.js";
+export { wakuLightPush } from "./protocols/sender/lightpush/index.js";
```

Comment:
tho I understand why this structure was proposed I think we should move into different direction making it leaner. I propose to have following for `sdk/src`:
```
src
- filter
  - index.ts
  - constants.ts
- lighPush
  - index.ts
- store
  - index.ts
- messageMonitor
  - index.ts
  - manager.ts
  - recceiverMonitor.ts
  - senderMonitor.ts
- waku.ts
- anything else
```

This is to ensure:
- imports are short;
- no one item namespaces;
- we follow module structure where folder is an entity and mostly has internal code and injected external code + some imports (utils / interfaces) mostly;